lms/static/styles/lms.scss

```diff
@@ -1,16 +1,22 @@
 // CSS entry point for back-end app
 @use 'normalize.css/normalize';
 
-@use 'variables' as var;
+$color-text: #202020;
+
+// Typography.
+$sans-font-family: 'Helvetica Neue', Helvetica, Arial, 'Lucida Grande',
+  sans-serif !default;
+
+$normal-font-size: 13px;
```

Comment:
👀 - I wonder if we could just eliminate this and let the font size go up to the 16px default.

Reply:
That could have some impact on UI, so I don't want to do it blindly!